dev-util/bpftool/bpftool-9999.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LLVM_COMPAT=( {15..19} )
+LLVM_OPTIONAL=1
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit bash-completion-r1 linux-info llvm-r1 optfeature python-any-r1 toolchain-funcs
+
+DESCRIPTION="Tool for inspection and simple manipulation of eBPF programs and maps"
+HOMEPAGE="https://github.com/libbpf/bpftool"
+
+if [[ ${PV} == *9999* ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/libbpf/bpftool.git"
+	EGIT_SUBMODULES=(libbpf)
+else
+	# uncomment to bundle an updated libbpf
+	# LIBBPF_VERSION=1.5.0
+
+	if [ ! -z ${LIBBPF_VERSION} ] ; then
+		SRC_URI="https://github.com/libbpf/bpftool/archive/refs/tags/v${PV}.tar.gz -> bpftool-${PV}.tar.gz
+			https://github.com/libbpf/libbpf/archive/refs/tags/v${LIBBPF_VERSION}.tar.gz
+			  -> libbpf-${LIBBPF_VERSION}.tar.gz"
+	else
+		# use tarball with bundled libbpf
+		SRC_URI="https://github.com/libbpf/bpftool/releases/download/v${PV}/bpftool-libbpf-v${PV}-sources.tar.gz"
+		S="${WORKDIR}/bpftool-libbpf-v${PV}-sources"
+	fi
+
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="caps llvm"
+
+RDEPEND="
+	caps? ( sys-libs/libcap:= )
+	llvm? ( $(llvm_gen_dep 'sys-devel/llvm:${LLVM_SLOT}') )
+	!llvm? ( sys-libs/binutils-libs:= )
+	sys-libs/zlib:=
+	virtual/libelf:=
+"
+DEPEND="
+	${RDEPEND}
+	>=sys-kernel/linux-headers-5.8
+"
+BDEPEND="
+	${PYTHON_DEPS}
+	app-arch/tar
+	dev-python/docutils
+"
+
+CONFIG_CHECK="~DEBUG_INFO_BTF"
+
+pkg_setup() {
+	python-any-r1_pkg_setup
+	use llvm && llvm-r1_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	# prepare libbpf if necessary
+	if [ ! -z ${LIBBPF_VERSION} ] ; then
```

Comment:
Please use Bash tests (so `[[` and `]]`) per https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101.

Reply:
Updated, thanks.